.github/workflows/swiftlint.yml

```diff
@@ -20,4 +20,4 @@ jobs:
         - uses: actions/checkout@v4
         - name: GitHub Action for SwiftLint
           run: |
-            swiftlint --reporter github-actions-logging --strict
+            swiftlint --reporter github-actions-logging --strict --quiet
```

Comment:
Why is the `--quiet` needed?
Will GitHub still show the linting errors in the diff?

Reply:
Yes. `--quite` just removed the noise for all files etc.. you can check the runs here:
<img width="995" alt="Screenshot 2024-02-12 at 10 57 13" src="https://github.com/wireapp/wire-ios/assets/7196958/b858118b-980b-4bf7-8890-8e57cbdf6609">

<img width="725" alt="Screenshot 2024-02-12 at 10 57 54" src="https://github.com/wireapp/wire-ios/assets/7196958/ff26708e-20b0-4c4a-9a5f-93c2ea52982c">